src/ext/state_context.rs

```diff
@@ -2,7 +2,8 @@ use crate::{cstr, plugin::*, stream::*};
 
 use std::ffi::CStr;
 
-pub const CLAP_EXT_STATE_CONTEXT: &CStr = cstr!("clap.state-context.draft/1");
+pub const CLAP_EXT_STATE_CONTEXT: &CStr = cstr!("clap.state-context/1");
```

Comment:
This should be `"clap.state-context/2"`: https://github.com/free-audio/clap/compare/1.1.10..1.2.0#diff-b95d5c5682aecd38a3e93eff208f59826ae0ad26e7c043b232d07dd4f817c869R35

Reply:
And it looks like there is no `CLAP_EXT_STATE_CONTEXT_COMPAT` in the upstream headers.